tests/memfile/run-test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
```

Comment:
@eddiehung FYI: I added this and two more checks at the bottom of the file (for situations where the execution must fail).

Reply:
@eddiehung it says that some checks haven't completed but if you enter to the details all the jobs had finished. There was a problem getting an Ubuntu package, it was re-launched, passed, but is marked still as pending.